CMakeLists.txt

```diff
@@ -0,0 +1,20 @@
+project(TOY_MT_FRAMEWORK)
+cmake_minimum_required(VERSION 2.8)
+
+set(CMAKE_MODULE_PATH ${CMAKE_MODULE_PATH} "${CMAKE_CURRENT_SOURCE_DIR}/cmake/Modules")
+find_package(OpenMP)
+find_package(TBB)
+find_package(CppUnit)
+find_package(Boost)
+
+set(CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS} -std=c++0x")
+
+add_subdirectory(BusyWaitCalibration)
+add_subdirectory(DispatchProcessingDemo)
```

Comment:
Given I decided not to pursue this technology, I would be fine with only having this build on OS X.